sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/AuthenticationRegionInfo.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.specialized.cryptography;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class AuthenticationRegionInfo {
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
we shouldn't use jackson directly. we should use com.azure.core.util.serializer.ObjectSerializer instead.

Reply:
It looks like this can just be removed as it isn't used in the file